src/nidmm/metadata/attributes_addon.py

```diff
@@ -0,0 +1,36 @@
+# These dictionaries are applied to the generated attributes dictionary at build time
+# Any changes to the API should be made here. attributes.py is code generated
+
+attributes_obsolete = {
```

Comment:
I think it's an odd variable name. Sure, _today_ we only have obsolete attributes and we are marking them an "no". But tomorrow we'll be adding a "is_obsolete":True flag to things we obsolete after the Python API's firs "ship".

I'd change to attributes_codegene_method and document that we are not generating attributes found to be obsolete before the _first_ "release" of the Python API.

Reply:
Changed